src/components/Modal/Modal.tsx

```diff
@@ -1,26 +1,28 @@
-import { FormEventHandler, useState } from "react";
+import { FormEventHandler } from "react";
+import closeIcon from "../../assets/close-icon.svg";
```

Comment:
because it's using SVG file, actually no need to import

just use regular `img src="/assets/icon.svg"` below